doc/source/release/1.14.0-notes.rst

```diff
@@ -49,49 +67,119 @@ New features
+- Some minor performance improvements for inverting of ``Rotation`` objects.
 
 `scipy.special` improvements
 ============================
+- Added `scipy.special.log_wright_bessel`, for calculation of the logarithm of
+  Wright's Bessel function.
+- Generalized ufunc support has been expanded.
+- The relative error in `scipy.special.hyp2f1` calculations has improved
+  substantially.
+- Improved behavior of ``boxcox``, ``inv_boxcox``, ``boxcox1p``, and
+  ``inv_boxcox1p`` by preventing premature overflow.
 
 
 `scipy.stats` improvements
 ==========================
-
-Hypothesis Tests
-----------------
-
-
-Sample statistics
------------------
-
-
-Statistical Distributions
--------------------------
-
-
-Other
------
+- A new function `scipy.stats.power` can be used for simulating the power
+  of a hypothesis test with respect to a specified alternative.
+- The Irwin-Hall (AKA Uniform Sum) distribution has been added as
+  `scipy.stats.irwinhall`.
+- Exact p-value calculations of `scipy.stats.mannwhitneyu` are much faster
+  and use less memory.
+- `scipy.stats.pearsonr` now accepts n-D arrays and computes the statistic
+  along a specified ``axis``.
+- `scipy.stats.kstat`, `scipy.stats.kstatvar`, and `scipy.stats.bartlett`
+  are faster at performing calculations along an axis of a large n-D array.
 
 
+**************************
+Array API Standard Support
+**************************
+*Experimental* support for array libraries other than NumPy has been added to
+existing sub-packages in a recent version of SciPy. Please consider testing
+these features by setting an environment variable ``SCIPY_ARRAY_API=1`` and
+providing PyTorch, JAX, or CuPy arrays as array arguments.
+
+From previous releases:
+
+- `scipy.cluster`
+- `scipy.fft`
+
+New in this release:
+
+- `scipy.constants`
+- `scipy.stats` (partial support):
+
+  - `scipy.stats.moment`
+  - `scipy.stats.skew`
+  - `scipy.stats.kurtosis`
+  - `scipy.stats.kstat`
+  - `scipy.stats.kstatvar`
+  - `scipy.stats.circmean`
+  - `scipy.stats.circvar`
+  - `scipy.stats.circstd`
+  - `scipy.stats.entropy`
+  - `scipy.stats.variation`
+  - `scipy.stats.sem`
+  - `scipy.stats.ttest_1samp`
+  - `scipy.stats.pearsonr`
+  - `scipy.stats.chisquare`
+  - `scipy.stats.skewtest`
+  - `scipy.stats.kurtosistest`
+  - `scipy.stats.normaltest`
+  - `scipy.stats.jarque_bera`
+  - `scipy.stats.bartlett`
+  - `scipy.stats.power_divergence`
+  - `scipy.stats.monte_carlo_test`
 
 
 *******************
 Deprecated features
 *******************
+- `scipy.stats.gstd`, `scipy.stats.chisquare`, and
+  `scipy.stats.power_divergence` have deprecated support for masked array
+  input.
+- The ``conjtransp`` method for `scipy.sparse.dok_array` and
+  `scipy.sparse.dok_matrix` has been deprecated and will be removed in SciPy
+  1.16.0.
+- The option ``quadrature="trapz"`` in `scipy.integrate.quad_vec` has been
+  deprecated in favour of ``quadrature="trapezoid"`` and will be removed in
+  SciPy 1.16.0.
```

Comment:
```suggestion
  SciPy 1.16.0.
- `scipy.special.comb` has deprecated support for non-integer arguments `N` and `k`.
```
gh-20780

Reply:
Thanks. I wrote the following in the wikipage:
```
- Setting ``exact=True`` when passing non-integer inputs to ``special.comb`` is deprecated (it has no effect).
```
Either is fine to me.